components/brave_rewards/core/endpoints/get_wallet/get_wallet.h

```diff
@@ -58,6 +59,10 @@ class GetWallet final : public RequestBuilder,
   ~GetWallet() override;
 
  private:
+  const char* Path() const;
+
+  absl::optional<std::vector<std::string>> Headers(
+      const std::string& content) const override;
   absl::optional<std::string> Url() const override;
```

Comment:
Nit: can we go in `RequestBuilder`'s declaration order (`Url()`, `Headers()`)?

Reply:
Sure, fixed.

Reply:
Also the implementations, thanks! 🙂

Reply:
Ah yes, fixed!